pyproject.toml

```
@@ -9,6 +9,9 @@ readme = "README.md"
 [tool.poetry.dependencies]
 python = "^3.11"
```

Based on the best practices when we created toy package, would it make sense for the version to be 3.11 or greater?

```suggestion
python = ">=3.11"
```

---

Yeah I think it's better to specify the python version as such.